specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/payments.json

```diff
@@ -0,0 +1,572 @@
+        },
+        "family": {
+          "description": "The family of payment method.",
+          "enum": [
+            "CreditCard",
+            "CheckWire"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "paymentMethodFamily"
+          }
+        },
+        "id": {
+          "description": "Id of payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "lastFourDigits": {
+          "description": "Last four digits of payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "logos": {
+          "description": "The list of logos for the payment method.",
+          "items": {
+            "$ref": "#/definitions/PaymentMethodLogo"
+          },
+          "type": "array"
+        },
+        "status": {
+          "description": "Status of the payment method.",
+          "enum": [
+            "active",
+            "inactive",
+            "unknown"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "paymentMethodStatus"
+          }
+        },
+        "type": {
+          "description": "The type of payment method.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "PaymentMethodProperties": {
+      "description": "The properties of a payment method.",
+      "properties": {
+        "accountHolderName": {
+          "description": "The account holder name for the payment method. This is only supported for payment methods with family CreditCard.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "displayName": {
+          "description": "The display name of the payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "expiration": {
+          "description": "The expiration month and year of the payment method. This is only supported for payment methods with family CreditCard.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "family": {
+          "description": "The family of payment method.",
+          "enum": [
+            "CreditCard",
+            "CheckWire"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "paymentMethodFamily"
+          }
+        },
+        "lastFourDigits": {
+          "description": "Last four digits of payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "logos": {
+          "description": "The list of logos for the payment method.",
+          "items": {
+            "$ref": "#/definitions/PaymentMethodLogo"
+          },
+          "type": "array"
+        },
+        "status": {
+          "description": "Status of the payment method.",
+          "enum": [
+            "active",
```

Comment:
Same here with the fallback for active/inactive... Also, I notice that the casing for enum values is not consistent. Can we make them consistant?

Reply:
made all lower case